emitter/ingress/values.yaml

```diff
@@ -0,0 +1,66 @@
+# Default values for emitter.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: nginx
```

Comment:
Can you give a bit more detail on why is Nginx being added? We already have Traefik deployed after @c0dyhi11 changed to use k3s. We can't have both in the cluster.